pulp_python/app/models.py

```diff
@@ -152,11 +152,11 @@ def __str__(self):
         e.g. <PythonPackageContent: shelf-reader [version] (whl)>
 
         """
-        return '<{obj_name}: {name} [{version}] ({type})>'.format(
+        return '<{obj_name}: {name} [{version}] ({_type})>'.format(
```

Comment:
This actually doesn't need to be changed, it maps to packagetype not the underlying "_type" field.